modules/howtos/pages/analytics-using-sdk.adoc

```diff
@@ -81,9 +81,9 @@ include::example$analytics.go[tag=options]
 
 == Handling the Response
 
-These query result may contain various sorts of data and metadata, 
-depending upon the nature of the query, 
-as you will have seen when working through our xref:6.5@server:analytics:primer-beer.adoc[introductory primer]. 
+These query result may contain various sorts of data and metadata,
```

Comment:
Do we need `results` here maybe?

Reply:
Oops, sloppy of me to include this whitespace change in commit. I'll do a driveby commit for the plural, good spot!

Reply:
Fixed in commit below!